third_party/libevent/evutil.h

```diff
@@ -38,7 +38,12 @@
 extern "C" {
 #endif
 
+#if defined LIBEVENT_CONFIG_PLATFORM_HEADER
```

Comment:
This is a better approach, but maybe it fits better _inside_ of the "event-config.h" at the top before the other conditions ?

That change could even be sent to upstream and be accepted, i think ?

Reply:
For what it's worth, it's not really an upstreaming concern for us much longer, Chromium removed `libevent` entirely: https://issues.chromium.org/issues/40057013